src/components/AutoComplete/AutoComplete.stories.js

```diff
@@ -0,0 +1,96 @@
+
+AutoCompleteStories.addDecorator(withKnobs);
+
+AutoCompleteStories.addWithInfo('Basic Example', () => {
+  const highlightOnlyResult = boolean('highlight Only Result', true);
+  const clearButton = boolean('Clear Button', true);
+  const multiple = boolean('Multiple Selection', false);
+  const allowNew = boolean('Allow Adding New Items', false);
+  const selectHintOnEnter = boolean('select Hint On Enter', true);
+  const minLength = number("Min Length to Autocomplete", 0);
+  const bsSize = select('Size', [undefined, 'default', 'small', 'large']);
+  const align = select('Dropdown Alignment', [undefined, 'justify', 'left', 'right']);
+  const props = { bsSize, align, clearButton, multiple, allowNew, options, minLength, selectHintOnEnter, highlightOnlyResult }
```

Comment:
Those props aren't relevant to the other stories?

Reply:
I am not sure if I want to show all of the knobs for each story.. On the other hand, there is a chance that someone who skipped the basic example won't understand how to use this props.